src/test/java/com/exasol/adapter/document/files/extension/ExtensionManagerInstaller.java

```diff
@@ -7,51 +7,82 @@
 
```

Comment:
Maybe we should spend a dedicated package "installer" here and create 3 files? 
* interface ExtensionManagerInstaller 
* GoInstaller 
* LocalProjectDirInstaller

Reply:
✅